articles/defender-for-cloud/enhanced-security-features-overview.md

```diff
@@ -121,7 +121,7 @@ When you enable the Servers plan on the subscription level, Defender for Cloud w
 
 However, if you're using a custom workspace in place of the default workspace, you'll need to enable the Servers plan on all of your custom workspaces that don't have it enabled. 
 
-If you're using a custom workspace and enable the plan on the subscription level only, the `Microsoft Defender for servers should be enabled on workspaces` recommendation will appear on the Recommendations page. This recommendation will give you the option to enable the servers plan on the workspace level with the Fix button. You're charged for all VMs in the subscription even if the Servers plan isn't enabled for the workspace. The VMs won't benefit from features that depend on the Log Analytics workspace, such as Microsoft Defender for Endpoint, VA solution (TVM/Qualys), and Just-in-Time VM access.
+If you're using a custom workspace and enable the plan on the subscription level only, the `Microsoft Defender for servers should be enabled on workspaces` recommendation will appear on the Recommendations page. This recommendation will give you the option to enable the servers plan on the workspace level with the Fix button. You're charged for all VMs in the subscription even if the Servers plan isn't enabled for the workspace. The VMs won't benefit from features that depend on the Log Analytics workspace, such as Adaptive Application Controls, File Integrity Monitoring, or the 500-MB free data ingestion allowance.
```

Comment:
```suggestion
If you're using a custom workspace and enable the plan on the subscription level only, the `Microsoft Defender for servers should be enabled on workspaces` recommendation will appear on the Recommendations page. This recommendation will give you the option to enable the servers plan on the workspace level with the Fix button. You're charged for all VMs in the subscription even if the Servers plan isn't enabled for the workspace. The VMs won't benefit from features that depend on the Log Analytics workspace, such as Adaptive Application Controls, File Integrity Monitoring, SQL Protection, ProtectionStatus reports from the Endpoint Protection Antimalware or the 500MB free data ingestion allowance.
```

Reply:
Hi @helderpinto I checked in with some local CxE people and they absolutely agreed with your change. They also suggested this small addition, let me know what you think and we can publish this ASAP.

Reply:
Hi @ElazarK. I do agree with the small (but very important) addition. I've just committed your suggestion. Thanks for the review!